src/data/general-cs-quiz.ts

```diff
@@ -934,6 +934,24 @@ const generalCSQuiz = [
     Explanation: "Merge sort is a divide and conquer algorithm that has a best case time complexity of O(n*log n)",
     Link: "https://www.freecodecamp.org/news/an-intro-to-advanced-sorting-algorithms-merge-quick-radix-sort-in-javascript-b65842194597/",
   },
+  {
+    Question: "What is the advanatge of using linked lists over arrays?",
+    Answer: "Linked lists helps in faster insertions and deletions than arrays",
+    Distractor1: "Memory usage in linked lists is less than arrays",
+    Distractor2: "Reverse traversal is possible in singly linked lists",
+    Distractor3: "Random accessing elements is easier in linked lists than arrays",
+    Explanation: "In Linked lists there is no requirement to shift all other elements while doing insertions or deletions, only address gets updated.",
+    Link: "https://www.freecodecamp.org/news/data-structures-101-linked-lists-254c82cf5883/",
+  },
+  {
+    Question: "Which sorting algorithm is best when data is nearly sorted?",
+    Answer: "Bubble Sort",
+    Distractor1: "Merge Sort",
+    Distractor2: "Insertion Sort",
+    Distractor3: "Selection Sort",
+    Explanation: "If only few elements are not sorted then Bubble Sort is the best sorting algorithm as it will sort the data with only few passes of the array.",
```

Comment:
```suggestion
    Explanation: "If the list is nearly sorted, then Bubble Sort is a good choice because it will sort the data with only few passes of the array.",
```